src/test/scala/org/graphframes/GraphFrameTestSparkContext.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.spark.sql.SQLContext
 trait GraphFrameTestSparkContext extends BeforeAndAfterAll { self: Suite =>
   @transient var sc: SparkContext = _
   @transient var sqlContext: SQLContext = _
+  @transient var spVers: Seq[Int] = _
```

Comment:
add the integer version fields sequence so that we can differentiate test cases based on `Spark` versions